src/java/README.md

```diff
@@ -20,7 +20,9 @@ Installs Java, SDKMAN! (if not installed), and needed dependencies.
 | version | Select or enter a Java version to install | string | latest |
 | jdkDistro | Select or enter a JDK distribution | string | ms |
 | installGradle | Install Gradle, a build automation tool for multi-language software development | boolean | false |
+| gradleVersion | Select a Gradle version | string | latest |
```

Comment:
Just fyi this will be auto-generated by [an Action](https://github.com/devcontainers/features/blob/main/.github/workflows/update-documentation.yml), but thank you for thinking to update :)

Reply:
I have seen it after pushing the PR so I kept it, thanks for the tip :)